openpype/hosts/houdini/startup/MainMenuCommon.xml

```diff
@@ -14,18 +14,18 @@ host_tools.show_creator(parent)
 ]]></scriptCode>
             </scriptItem>
 
-            <scriptItem id="avalon_load">
```

Comment:
You have removed loader...

Reply:
huh.. perhaps I felt it was not useful anymore. Joking, thanks - nice catch! ;)